### ISSUE TITLE
BUG - no-unused doesn't recognize React imports for JSX

### ISSUE THREAD

**Issue by felixmosh** [ajafff/tslint-consistent-codestyle#38]
For a React import that needed only for JSX, `no-unused` throws an error `ERROR: src/app.tsx[1, 13]: Import 'React' is unused.`

```javascript
import * as React from 'react';
import * as ReactDom from 'react-dom';

ReactDom.render(<App />, document.getElementById('root'));
```

As a temporary workaround, we can set `ignore-imports` to true.

**Comment by ajafff** [comment]
Fix is release in v1.10.1

**Comment by felixmosh** [comment]
Thanx!
